core/modules/rewrite.cc

```diff
@@ -107,7 +107,7 @@ inline void Rewrite::DoRewrite(bess::PacketBatch *batch) {
 
   next_turn_ = start + cnt;
   if (next_turn_ >= bess::PacketBatch::kMaxBurst) {
-    next_turn_ -= bess::PacketBatch::kMaxBurst;
+    next_turn_ = next_turn_ % num_templates_;
```

Comment:
The hack of using `kNumSlots` was to avoid integer modulo instruction, which used to appear significant overhead when profiled. Perhaps we can achieve the fairness without using `%`? (e.g. lookup table)